scripts/depcheck.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Why do we need this shell script? Can we just call depcheck directly?

Reply:
Depcheck returns a bunch of other things, such as dependencies that are imported but not used, and apparently this is not 100% accurate so I did a little bit of checking for only un-imported dependencies. For instance, it does not think we are using jest.

Reply:
For instance, right now it's returning this:

```
Unused devDependencies
* @types/jest
```

which I think is safe to ignore because we shouldn't care too much about unused dependencies as they're not going to break anything

Reply:
Are we in fact not using that devDependency? Can we remove it? I think it would be cleaner to fix any issues depcheck finds rather than adding a wrapper around it. On the other hand, if depcheck is giving inaccurate results that is concerning 🤔

Reply:
If there's some issue specifically around Jest, it looks like depcheck has an `--ignores` option which might work as well

Reply:
Hmm interesting, that's a good catch. I did end up removing it and it seems to be passing. I had thought that I tested this and the tests had failed but that might have been an unrelated issue. Apparently there can be false positives, though. I've fixed the issue and tested it, should be a lot cleaner now.

> The logic of a special is not perfect. There might be false alerts. If this happens, please open an issue for us.

Reply:
Okay yeah, it looks like the tests are failing in CI now but not locally after removing `@types/jest`. Not really sure what to do here.

Reply:
I'm just going to ignore `@types/jest` for now.